python-client/tests/integrations/test_wandb.py

```diff
@@ -2,25 +2,36 @@
 import pytest
 import re
 
-from giskard import scan
+import numpy as np
+
+from giskard import scan, Dataset
+from giskard.models.model_explanation import explain_with_shap, explain_full, explain_one
 
 wandb.setup(wandb.Settings(mode="disabled", program=__name__, program_relpath=__name__, disable_code=True))
 
 
 @pytest.mark.parametrize(
     "dataset_name,model_name",
     [
+        ("hotel_text_data", "hotel_text_model"),
         ("german_credit_data", "german_credit_model"),
         ("breast_cancer_data", "breast_cancer_model"),
         ("drug_classification_data", "drug_classification_model"),
         ("diabetes_dataset_with_target", "linear_regression_diabetes"),
-        ("hotel_text_data", "hotel_text_model"),
     ],
 )
 def test_fast(dataset_name, model_name, request):
+    # Expect the 'NotImplementedError' when dataset contains textual features.
+    exception_fixtures = ("hotel_text_data",)
+
     dataset = request.getfixturevalue(dataset_name)
     model = request.getfixturevalue(model_name)
-    _to_wandb(model, dataset)
+    _compare_explain_functions(model, dataset)
+
+    if dataset_name in exception_fixtures:
+        with pytest.raises(NotImplementedError) as e:
+            _to_wandb(model, dataset)
+        assert e.match(r"We do not support the SHAP logging of text*")
```

Comment:
there should be an else here. 
```py
else:
  _to_wandb(model, dataset)
```
the main point was to check if `to_wandb` works when there's no text features.

Reply:
Done